src/Client/Core/OrchestrationMetadata.cs

```diff
@@ -1,233 +1,238 @@
+/// </remarks>
+public sealed class OrchestrationMetadata
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="OrchestrationMetadata" /> class.
+    /// </summary>
+    /// <param name="name">The name of the orchestration.</param>
+    /// <param name="instanceId">The instance ID of the orchestration.</param>
+    public OrchestrationMetadata(string name, string instanceId)
+    {
+        this.Name = name;
+        this.InstanceId = instanceId;
+    }
+
+    /// <summary>Gets the name of the orchestration.</summary>
+    /// <value>The name of the orchestration.</value>
+    public string Name { get; }
+
+    /// <summary>Gets the unique ID of the orchestration instance.</summary>
+    /// <value>The unique ID of the orchestration instance.</value>
+    public string InstanceId { get; }
+
+    /// <summary>
+    /// Gets the data converter used to deserialized the serialized data on this instance.
+    /// This will only be present when inputs and outputs are requested, <c>null</c> otherwise.
+    /// </summary>
+    /// <value>The optional data converter.</value>
+    public DataConverter? DataConverter { get; init; }
+
+    /// <summary>
+    /// Gets the current runtime status of the orchestration instance at the time this object was fetched.
+    /// </summary>
+    /// <value>The runtime status of the orchestration instance at the time this object was fetched.</value>
+    public OrchestrationRuntimeStatus RuntimeStatus { get; init; }
+
+    /// <summary>
+    /// Gets the orchestration instance's creation time in UTC.
+    /// </summary>
+    /// <value>The orchestration instance's creation time in UTC.</value>
+    public DateTimeOffset CreatedAt { get; init; }
+
+    /// <summary>
+    /// Gets the orchestration instance's last updated time in UTC.
+    /// </summary>
+    /// <value>The orchestration instance's last updated time in UTC.</value>
+    public DateTimeOffset LastUpdatedAt { get; init; }
+
+    /// <summary>
+    /// Gets the orchestration instance's serialized input, if any, as a string value.
+    /// </summary>
+    /// <value>The serialized orchestration input or <c>null</c>.</value>
+    public string? SerializedInput { get; init; }
+
+    /// <summary>
+    /// Gets the orchestration instance's serialized output, if any, as a string value.
+    /// </summary>
+    /// <value>The serialized orchestration output or <c>null</c>.</value>
+    public string? SerializedOutput { get; init; }
+
+    /// <summary>
+    /// Gets the orchestration instance's serialized custom status, if any, as a string value.
+    /// </summary>
+    /// <value>The serialized custom status or <c>null</c>.</value>
+    public string? SerializedCustomStatus { get; init; }
+
+    /// <summary>
+    /// Gets the tags associated with the orchestration instance.
+    /// </summary>
+    public IReadOnlyDictionary<string, string> Tags { get; init; } = new Dictionary<string, string>();

```

Comment:
Remove the `init` since we're providing the underlying dictionary.
```suggestion
    public IReadOnlyDictionary<string, string> Tags { get; } = new Dictionary<string, string>();
```

Reply:
Fixed but I can't use IReadOnlyDictionary

Reply:
fixed